src/v/security/audit/audit_log_manager.cc

```diff
@@ -296,15 +306,16 @@ ss::future<> audit_client::shutdown() {
     if (_as.abort_requested()) {
         co_return;
     }
+    _probe.reset(nullptr);
```

Comment:
I guess the same cannot be said here since `_probe` is never used in any background loops, but if someone in the future does add it to work executing within the context of a gate it will be a problem, i would just omit this line or again move it to the bottom of the method after the gate closes.